packages/cms-cli/commands/init.js

```diff
@@ -55,13 +58,21 @@ const AUTH_METHOD_FLOW = {
     },
     setup: async configData => {
       createEmptyConfigFile();
-      process.on('exit', deleteEmptyConfigFile);
+      handleExit();
       await authenticateWithOauth(configData);
+      logger.log(
+        `Success: ${DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME} created with ${AUTH_METHODS.oauth.name}.`
+      );
       process.exit();
     },
   },
 };
 
+const handleExit = () => {
+  process.on('exit', deleteEmptyConfigFile);
+  process.on('SIGINT', deleteEmptyConfigFile);
+};
```

Comment:
Isn't there a hoisting issue with this down here?

Reply:
It works because it's called within a function that is the value of an object property that is declared above this code. I can move it up to avoid any confusion though.